From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 14 Mar 2016 17:47:16 +0000
Subject: Revert "libata: Align ata_device's id on a cacheline"
Forwarded: not-needed

This reverts commit cea2cbff57c5b65375adb6fe65612c10a7301327, which
was commit 4ee34ea3a12396f35b26d90a094c75db95080baa upstream.  It's a
real fix but we've lived with the problem for many years and it
doesn't seem worth breaking ABI to fix it right now.
---
 include/linux/libata.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/libata.h
+++ b/include/linux/libata.h
@@ -718,7 +718,7 @@ struct ata_device {
 	union {
 		u16		id[ATA_ID_WORDS]; /* IDENTIFY xxx DEVICE data */
 		u32		gscr[SATA_PMP_GSCR_DWORDS]; /* PMP GSCR block */
-	} ____cacheline_aligned;
+	};
 
 	/* DEVSLP Timing Variables from Identify Device Data Log */
 	u8			devslp_timing[ATA_LOG_DEVSLP_SIZE];
